src/main/java/org/openhab/binding/zwave/handler/ZWaveControllerHandler.java

```diff
@@ -653,17 +644,11 @@ protected void incomingMessage(SerialMessage serialMessage) {
 
     @Override
     public void deviceDiscovered(int nodeId) {
```

Comment:
Why not just delete this method?

Reply:
right, `ZWaveControllerHandler `is an abstract class, so it can be deleted here, but then I need to add it to `ZWaveSerialHandler`, because it extends `ZWaveControllerHandler `.
Is it better to do it that way?

Reply:
Well I think you need to understand why there is this method as you've stopped it doing anything. If I remember correctly it is part of the IoHandler interface so if it's not used, it should be removed - if it is used, it should be used?

Reply:
Hm, it doesn't do anything if I see it correctly.
The call to the discoveryService in `ZWaveControllerHandler` was commented out before and there is no other implementation of this method.
DeviceDiscovery is handled via events in initial and later stage, so there is no need, as far as I can see it.

So I would propose to remove the method in the IoHandler interface.
I did that and all tests run successfully.

Should I add this change to the pull?

Reply:
Thanks. Yes, this is what I was aiming for :)